src/components/Navigation.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import { FaUserCircle } from 'react-icons/fa';
+
+const Navigation = () => {
```

Comment:
- [ ] I think it would be better if styling is exactly the same as the one provided on [Zeplin](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/sneak_peek.md) because this means you do your job professionally by paying attention to even details. Let's compare yours and Zeplin design and fix the differences 👇 
- Your header below 👇 
![student-1](https://user-images.githubusercontent.com/67087664/151560980-0101d8b5-c1b9-4652-a5fe-5c8c9e20ff49.jpg)
- Zeplin header 👇 
![zeplin-1](https://user-images.githubusercontent.com/67087664/151561065-f2d51fd8-1266-46ce-9be6-96cc428fa660.jpg)
- Your booklist 👇 
![student-2](https://user-images.githubusercontent.com/67087664/151561217-7fda3e78-aab6-481a-a247-8848d45c69bb.jpg)
- Zeplin 👇 
![zeplin-2](https://user-images.githubusercontent.com/67087664/151561254-954523c6-b8d5-4c79-bfce-58d6f5d8a31f.jpg)
I have marked the differences with yellow color, please kindly use similar colors, space, margins, paddings, etc.

Reply:
- [ ] The styling issues here have not been properly resolved. Kindly use similar spacing, margins and padding as the zeplin designs.